app/src/main/java/ar/scacchipa/twittercloneapp/data/repository/TweetRepository.kt

```diff
@@ -0,0 +1,42 @@
+package ar.scacchipa.twittercloneapp.data.repository
+
+import ar.scacchipa.twittercloneapp.data.IMapper
+import ar.scacchipa.twittercloneapp.data.datasource.ITweetExternalSource
+import ar.scacchipa.twittercloneapp.data.model.response.TweetsDataWrapper
+import ar.scacchipa.twittercloneapp.domain.model.ResponseDomain
+import ar.scacchipa.twittercloneapp.domain.model.TweetCardInfo
+import ar.scacchipa.twittercloneapp.utils.Constants
+
+class TweetRepository(
+    private val tweetExternalSource: ITweetExternalSource,
+    private val credentialRepository: ICredentialRepository,
+    private val ownerUserRepository: IOwnerUserRepository,
```

Comment:
This seems wired, this repo I think should not know other repos, coordinate repos is a use case task.

Reply:
Now, The TweetRepository has dependencies from SourceData. The get accessToken and get loggedUserIdwas moved to UseCase layer.